src/gateway/lambda_function.py

```diff
@@ -227,6 +227,50 @@ def _run_enumerate(arguments):
         ),
     )
 
+    try:
+        obj = s3c.get_object(Bucket="rpmrepo-storage", Key="data/meta/enumerate_cache.json")
```

Comment:
maybe check `obj.LastModified`? and report/update this somehow?

Reply:
hmm do you mean to only update the objects that were modified?